bindata/linuxptp/ptp-daemon.yaml

```diff
@@ -216,7 +216,19 @@ spec:
   groups:
     - name: ptp.rules
       rules:
+        {{ if (eq .EnableEventPublisher true) }}
         - alert: NodeOutOfPtpSync
+            annotations:
+              message: |
+                {{ "{{" }} $labels.iface }} is not in sync
+            expr: |
+              openshift_ptp_clock_state != 1
+            for: 2m
+            labels:
+              severity: warning
+        {{ end }}
+        {{ if (eq .EnableEventPublisher false) }}
```

Comment:
I have two conditions because with the event publisher the offset is not fixed to 100 nano sec. it is variable and openshift_ptp_clock_state will be set to 1 whenever the offset value is outside the threshold.
In this case, we don't want to alert for 100 nano sec offset.
In the regular path, we only fix it to 100 nano sec since there are no threshold values used. 
Unless we introduced threshold values for the regular PTP path too (which can be in 4.10 )